.github/workflows/PR_build_extension.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Copy language-server dist directory inside vscode package
         run: |
           cp -R packages/language-server packages/vscode
-          rm -R packages/vscode/language-server/src packages/vscode/language-server/test
+          rm -R packages/vscode/language-server/src packages/vscode/language-server/src/__test__/__fixtures__
```

Comment:
```suggestion
          rm -R packages/vscode/language-server/src packages/vscode/language-server/src/__test__/
```
Would be better?

Reply:
Sure! I was just trying to replicate the previous functionality but this makes sense too